src/include/liburing.h

```diff
@@ -739,6 +747,15 @@ static inline void io_uring_prep_renameat(struct io_uring_sqe *sqe, int olddfd,
 	sqe->rename_flags = (__u32) flags;
 }
 
+static inline void io_uring_prep_rename(struct io_uring_sqe *sqe,
```

Comment:
I also think it'd be cleaner if this one just called io_uring_prep_renameat(), and ditto for the other ones. Then we keep the RENAMEAT sqe fields consistent by using the same parent helper.